PendingReleaseNotes.md

```diff
@@ -3,3 +3,5 @@
 ## Breaking Changes
 
 ## Features
+
+* Add support for Ceph Tentacle(v20)
```

Comment:
Let's not add it to these pending release notes, as we will need to backport the support to 1.18.